src/Tribe/Metabox.php

```diff
@@ -904,6 +906,43 @@ public static function localize_decimal_character() {
 		) );
 	}
 
+	/**
+	 * Get the list panel helper text.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post $post The post object.
+	 *
+	 * @return string
+	 */
+	public function get_list_panel_helper_text( $post ) {
```

Comment:
Type-hint whenever possible.

```suggestion
	public function get_list_panel_helper_text( WP_Post $post ) : string {
```

Reply:
Why is this method  `public`? It looks like it should be `protected`.

Reply:
As we are returning a filtered value here, therefore, I have skipped the type hinting for the return type in the signature. 

Should we always do type hinting with filters as well?